lang/src/jmh/kotlin/org/partiql/jmh/benchmarks/ParserBenchmark.kt

```diff
@@ -0,0 +1,1368 @@
+                        max(thumb_image_width) as thumbnail_width, 
+                        max(media_duration) as duration, 
+                        case
+                            when mediaalbums.media_type = 1 and album_name = 'AudioFeeds'
+                            then 4
+                            when mediaalbums.media_type = 2 and album_name = 'VideoFeeds'
+                            then 5 else 8
+                        end as albumType, 
+                        count(mediacontents.id) as media_count,  
+                        media_artist as artist  
+                        FROM
+                        (mediaalbums INNER JOIN mediacontents ON mediaalbums.id = mediacontents.album_id)
+                        INNER JOIN newsfeedmediacontents
+                        ON newsfeedmediacontents.contentId = mediacontents.id group by newsfeedid 
+                        UNION 
+                        SELECT   
+                        -1 * userId as albumId,  
+                        newsFeedId,imageUrl as url, 
+                        max(imageHeight) as height, 
+                        max(imageWidth) as width, 
+                        (NULL) as thumnail_url, 
+                        (NULL) as thumnail_height, 
+                        (NULL) as thumbnail_width, 
+                        (NULL) as duration, 
+                        case
+                            when albumId = 'default' then 1
+                            when albumId = 'profileimages' then 2
+                            when albumId = 'coverimages' then 3
+                        end as albumType, 
+                        count(imageid) as media_count,  
+                        (NULL) as artist  
+                        FROM userimages
+                        INNER JOIN newsfeedimages on userimages.id = newsfeedimages.imageId
+                        group by newsfeedid 
+                    ) album 
+                ON newsfeed.id = album.newsfeedId 
+            ) 
+            LEFT JOIN 
+            ( 
+            select newsPortalFeedId as feedid,
+            title,description,shortDescription,newsUrl,externalUrlOption, newsPortalCategoryId as pagefeedcategoryid,
+            (15) as defaultCategoryType from newsportalFeedInfo 
+            UNION 
+            select businessPageFeedId as feedid,title,description,shortDescription,newsUrl,externalUrlOption,
+            businessPageCategoryId as pagefeedcategoryid,(25) as defaultCategoryType from businessPageFeedInfo 
+            UNION 
+            select newsfeedId as feedid,(NULL) as title,description,(NULL) as shortDescription,(NULL) as newsUrl,
+            (NULL) as externalUrlOption, categoryMappingId as pagefeedcategoryid,
+            (20) as defaultCategoryType from mediaPageFeedInfo 
+            ) page 
+            ON newsfeed.id = page.feedId WHERE privacy != 10 
+        """.trimIndent()
+
+        val queries = listOf(
```

Comment:
Nit: May be consider making this a map, so instead of using index, one can use variable name, which is actually meaningful here in this context.